.sync.yml

```diff
@@ -3,5 +3,4 @@
   secure: "fH1bR1eBgA9tztMHtgdBSEOhNChC2YyoAczMzjUWgfDjinU1WOUPEprfV9upf6ZeQSTxAES36BBOrdxB6BVgdXlvLlCID+Lge1fTfi75ZX34q+eTadig/iZp3M0OveL2b5OmDv0cGbmMq8ywhgG+GPFuwPy7HF1Qjgul8UGNFOs="
   docker_sets:
     - set: centos7-64
-    - set: debian8-64
```

Comment:
Could we update this to Debian 9 or is that a much bigger change?

Reply:
sadly debian isn't supported at all. But I will have a look after this is merged.